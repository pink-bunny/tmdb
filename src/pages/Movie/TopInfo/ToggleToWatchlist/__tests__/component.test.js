import React from 'react';
import { shallow } from 'enzyme';

import ToggleToWatchlistComponent from '../component';

describe('ToggleToWatchlistComponent component', () => {
  const requiredProps = {
    watchlist: false,
    changeWatchlistStatus: jest.fn()
  };
  /* eslint-disable react/jsx-props-no-spreading */
  const wrapper = shallow(<ToggleToWatchlistComponent {...requiredProps} />);
  /* eslint-enable react/jsx-props-no-spreading */

  it('matches snapshot with required props', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('matches snapshot whenwatchlist is turned on', () => {
    wrapper.setProps({ watchlist: true });

    expect(wrapper).toMatchSnapshot();
  });
});
