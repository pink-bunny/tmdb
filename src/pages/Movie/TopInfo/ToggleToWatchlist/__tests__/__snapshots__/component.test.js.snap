// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleToWatchlistComponent component matches snapshot whenwatchlist is turned on 1`] = `
<Icon
  onClick={[MockFunction]}
  theme="filled"
  type="book"
/>
`;

exports[`ToggleToWatchlistComponent component matches snapshot with required props 1`] = `
<Icon
  onClick={[MockFunction]}
  type="book"
/>
`;
